cmd/client/workflow.go

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright (C) 2018 Red Hat, Inc.
+ *
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package client
+
+import (
+	"os"
+
+	"github.com/robertkrimen/otto"
+
+	"github.com/skydive-project/skydive/api/client"
+	"github.com/skydive-project/skydive/api/types"
+	"github.com/skydive-project/skydive/common"
+	"github.com/skydive-project/skydive/js"
+	"github.com/skydive-project/skydive/logging"
+	"github.com/skydive-project/skydive/validator"
+
+	"github.com/spf13/cobra"
+)
+
+var (
+	workflowPath string
+)
+
+// WorkflowCmd workflow root command
+var WorkflowCmd = &cobra.Command{
+	Use:          "workflow",
+	Short:        "Manage workflows",
+	Long:         "Manage workflows",
+	SilenceUsage: false,
+}
+
+func loadWorklow(path string) (*types.Workflow, error) {
+	file, err := os.Open(workflowPath)
+	if err != nil {
+		return nil, err
+	}
+
+	workflow := &types.Workflow{}
+	if err := common.JSONDecode(file, workflow); err != nil {
```

Comment:
I would prefer to parse yaml instead of json so that the user can copy/paste an existing workflow to embedded ones

Reply:
Done